pkg/logger/logger.go

```diff
@@ -26,6 +33,23 @@ var (
 	customCallerEncoder = func(caller zapcore.EntryCaller, enc zapcore.PrimitiveArrayEncoder) {
 		enc.AppendString(colorYellow + caller.TrimmedPath() + colorReset)
 	}
+	customLevelEncoder = func(level zapcore.Level, enc zapcore.PrimitiveArrayEncoder) {
+		if level == zapcore.InfoLevel-1 {
+			enc.AppendString(colorOrange + "WARNING" + colorReset)
+		} else if level == zapcore.InfoLevel-2 {
+			enc.AppendString(colorPurple + "DEBUG" + colorReset)
+		} else if level < zapcore.InfoLevel-2 {
```

Comment:
do we need to support loglevel < zapcore.InfoLevel-2

Reply:
refer latest version